engines/bmc_simplepath.h

```diff
@@ -23,10 +23,10 @@ namespace pono {
 class BmcSimplePath : public KInduction
 {
  public:
-  BmcSimplePath(const Property & p, smt::SmtSolver & solver);
-  BmcSimplePath(const PonoOptions & opt,
-                const Property & p,
-                smt::SmtSolver & solver);
+  BmcSimplePath(const Property & p, smt::SolverEnum se);
+  BmcSimplePath(const Property & p, smt::SmtSolver solver);
+  BmcSimplePath(const PonoOptions & opt, const Property & p, smt::SolverEnum se);
+  BmcSimplePath(const PonoOptions & opt, const Property & p, smt::SmtSolver solver);
```

Comment:
const ref to smtSolver

Reply:
Thanks, addressed with 0dbc624d1d5d9ab57c7977b758211620fdff5ecd